master_write.py

```diff
@@ -0,0 +1,103 @@
+# 12/18/2020
+# by: Ryan T Debolt
+# go through each file, gather and plot/save important information.
+import csv
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import cm
+import matplotlib.colors as mcolors
+from cycler import cycler
+import statistics as stat
+from statistics import mean
+import numpy as np
+
+design = "PUEO"
+population = 50
+error = "0.5"
+
+count = 0
+earliest_gen = []
+average_gen = []
+average_gen2 = []
+sigma_average_gen = []
+sigma_average_gen2 = []
+run_name = []
+bench_mark = 0.5 ## the benchmark we are trying to be bellow in chi score
+bench_mark2 =0.25 
```

Comment:
```suggestion
bench_mark = 0.5  # the benchmark we are trying to be bellow in chi score
bench_mark2 = 0.25 
```
Comments usually have 2 spaces before the `#` (to further separate it from the code) and one after.